x-pack/plugins/lens/public/types.ts

```diff
@@ -25,6 +25,7 @@ import {
   VALUE_CLICK_TRIGGER,
   VisualizeFieldContext,
 } from '../../../../src/plugins/ui_actions/public';
+import { LensSortActionData, LENS_EDIT_SORT_ACTION } from './datatable_visualization/expression';
```

Comment:
This import is causing the current build to fail because all of the datatable vis is included in the initial bundle. This should be fixable by making it a type import: `import type { LensSortActionData, LENS_EDIT_SORT_ACTION } from './datatable_visualization/expression'; `